core/src/main/scala/com/nvidia/spark/rapids/tool/tuning/TunerContext.scala

```diff
@@ -53,7 +53,9 @@ case class TunerContext (
       appIndex: Int = 1,
       dsInfo: Seq[DataSourceProfileResult],
       platform: Platform): Option[TuningResult] = {
-    val rawAggMetrics = QualSparkMetricsAnalyzer.getAggRawMetrics(appInfo, appIndex)
+    val sqlAnalyzer = AppSQLPlanAnalyzer(appInfo, appIndex)
+    val rawAggMetrics =
+      QualSparkMetricsAnalyzer.getAggRawMetrics(appInfo, appIndex, Some(sqlAnalyzer))
```

Comment:
Are those diagnostics view supposed to be generated from within Qualification too?

Reply:
discussed this offline, we will keep these views in Qualification tool